lunes_cms/cms/views/feedback_viewset.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import mixins, viewsets
+from ..serializers import FeedbackSerializer
+
+
+class CreateFeedbackViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
+    """
+    A viewset that provides only the ``create`` action for feedback elements.
+    """
+
+    serializer_class = FeedbackSerializer
+    http_method_names = ["post"]
```

Comment:
```suggestion
```
This is already implicitly solved by only inheriting the `CreateModelMixin` and the `GenericViewSet`. All methods that don't create are automatically forbidden.